resources/migrations/000_migrations.yaml

```diff
@@ -8472,49 +8472,60 @@ databaseChangeLog:
                                   'DateTimeField', 'DecimalField', 'DictionaryField', 'FloatField', 'IntegerField',
                                   'TextField', 'TimeField', 'UUIDField', 'UnknownField');
   - changeSet:
-      id: 308
```

Comment:
This diff is trash. 306 was claimed but later not used, so I filled in the gap. 307+ are unaffected.